helpers/deploymentConfig.ts

```diff
@@ -35,13 +35,12 @@ export const preconfiguredAddresses = {
     LzVirtualChainId: "102",
   },
   sepolia: {
-    VTreasury: "0xFc43c055B9be2Ec3BEe6f8C291Af862d764016a0",
+    VTreasury: "0x4116CA92960dF77756aAAc3aFd91361dB657fbF8",
```

Comment:
This address can be imported from the npm package `@venusprotocol/venus-protocol`, v6.0.0-dev.3 at least